projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/aquatic/colors.scss

```diff
@@ -41,4 +41,11 @@ $color-primary-light: $studio-blue-30; // Calypso --color-primary-light
 $color-surface: $studio-white; // Calypso --color-surface
 $color-neutral-100-rgb: $studio-gray-100-rgb; // Calypso --color-neutral-100-rgb
 
+// Calypso Sidebar Nudges
+$menu-nudge-background: $studio-white;
+$menu-nudge-text-color: $studio-black;
+$menu-nudge-cta-background: $highlight-color;
+$menu-nudge-cta-color: $studio-white;
```

Comment:
Since these 4 variables have always the same value (except a very few themes), what do you think about moving them to `_variables.scss` file as defaults? That way we don't need to declare them on every single color scheme.

Reply:
`_variables.scss` is only loaded for Calypso Color schemes through `_admin.scss`.
https://github.com/Automattic/jetpack/pull/19773#discussion_r631137688

Loading `_variables.scss` inside each WP Admin color-scheme would be a solution, but I wouldn't like to explore the possible complications in this PR.